src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,33 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 16px;
+  margin-bottom: 24px;
+
+  &__container {
+    display: flex;
+  }
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &__rewiew {
+    font-size: 10px;
+    line-height: 14px;
+    color: $main-color;
+  }
+}
+
+.star-silver {
```

Comment:
<img width="830" alt="Screenshot 2023-08-10 at 16 41 43" src="https://github.com/mate-academy/layout_product-cards/assets/53228259/2f06efcd-4b3b-400e-898d-031fe4ae577c">

check it